persist/boltdb_test.go

```diff
@@ -196,9 +202,8 @@ func TestOpenDatabase(t *testing.T) {
 // with the wrong filemode (< 0600), which should result in an os.ErrPermission
 // error.
 func TestErrPermissionOpenDatabase(t *testing.T) {
-	if testing.Short() {
-		t.SkipNow()
-	}
+	t.Skip("can't reproduce on Windows")
```

Comment:
I think you were right about the directory. Files on Windows seem to inherit permissions from their parent directory. However, if we restrict permissions on the folder (i.e making it read-only), then we can no longer create files inside it!

I'm just gonna throw an OS guard around it and call it a day.